src/ProducerConfig.cc

```diff
@@ -33,6 +33,11 @@ static const std::string CFG_COMPRESS_TYPE = "compressionType";
 static const std::string CFG_BATCH_ENABLED = "batchingEnabled";
 static const std::string CFG_BATCH_MAX_DELAY = "batchingMaxPublishDelayMs";
 static const std::string CFG_BATCH_MAX_MSG = "batchingMaxMessages";
+static const std::string CFG_SCHEMA = "schema";
+static const std::string CFG_SCHEMA_TYPE = "schemaType";
+static const std::string CFG_SCHEMA_NAME = "name";
+static const std::string CFG_SCHEMA_SCHEMA = "schema";
+static const std::string CFG_SCHEMA_PROPS = "properties";
```

Comment:
```suggestion
static const std::string CFG_SCHEMA = "schema";
```

Reply:
Yup, thanks for the heads up on cleanup.

Reply:
Fixed in 7a29a67